src/api/app/components/card_component.rb

```diff
@@ -6,9 +6,10 @@ class CardComponent < ApplicationComponent
   renders_one :delete_button
   renders_many :actions
 
-  def initialize(extra_classes_for_body: [])
+  def initialize(extra_classes_for_card: [], extra_classes_for_body: [])
```

Comment:
💭 I don't dig this change. Passing around a string into a component every time to render it in the same place. Why?

Reply:
Former reason why I need it is that there are some `js` and `specs` relying on those classes (`repository-card` and `group-user`) so I cannot simply drop those classes.
Easiest solution would be to wrap every card by a `div` with the needed class, but that is something I want to avoid, and it would imply some `css` change anyway.
As we already have a typed wrapper container (`#tokens`, `#repositories`, `#group-users`) for all the groups of cards, ideally the best option would be to refactor `js` and `specs` in order to no longer rely on those classes. At this point I am not sure if it is in the scope of this PR or it is more a refactoring for later.

Reply:
I took the chance to refactor the code and drop the dependencies to those custom additional classes. Now there are no longer strings passed around: the component and the usages are is even more standardized and unified.

Thanks for raising it up.